src/models/user.ts

```diff
@@ -15,7 +15,7 @@ const userSchema = new Schema({
         type: String,
         default: 'user',
     },
-    profile: {
+    profile_id: {
```

Comment:
This field should remain as `profile` because when you call the user with the `populate` option, it is more meaningful by saying 
```
// get user with profile would be easier
const user = await User.findOne({where: {id:id}}).populate("profile", ["first_name",...])

```
Since we are not just storing the id despite the field type, we are referencing another object which is `profile` not `profile_id`